apps/aklite-offline/cmds.cpp

```diff
@@ -69,19 +69,19 @@ int RunCmd::runUpdate(const Config& cfg_in) const {
         break;
       }
       case offline::PostRunAction::RollbackOk: {
-        LOG_INFO << "Installation has failed so a device was rolled back to the previous version";
+        LOG_INFO << "Installation has failed so a device rollbacked to the previous version";
```

Comment:
"rolled back". we need to make the past tense of "roll" not "back".

Reply:
Quite expectedly, Oxford says that _rollback_ can only be used as a noun:
https://www.oxfordlearnersdictionaries.com/definition/english/rollback?q=rollback

Reply:
IIUC furthermore, the device is an object here, not a subject... thus a passive form used in the original _was rolled back_.

@kprosise what do you recommend.

Reply:
I made `device` a subject for purpose,  I think it's more correct.

Reply:
@mike-sul the purpose matters... I like that you gave it a thought ;)

Reply:
I'm enjoying seeing these sorts of conversations!